third_party/bazel/tf_source/no_tf_serving_workspace1.bzl.tpl

```diff
@@ -0,0 +1,10 @@
+def _repositories():
+    native.local_repository(
+        name = "org_tensorflow",
+        path = "../tf_community/",
+    )
+
+def workspace():
+    _repositories()
+
+tf_source_workspace1 = workspace
```

Comment:
nit: maybe `no_tf_xxx` is not a good enough name ?

Reply:
Maybe `tf_community_workspace.bzl` ?

Reply:
looks better :P